lib/existing-eks-opensource-observability-pattern/istio/istioIngressGatewayAddon.ts

```diff
@@ -0,0 +1,26 @@
+import 'source-map-support/register';
```

Comment:
Why do we need this file as duplicate. We can refer to file in Single-New folder?